prime-angular-frontend/src/app/modules/enrolment/shared/services/enrolment-form-state.service.ts

```diff
@@ -168,14 +170,36 @@ export class EnrolmentFormStateService extends AbstractFormStateService<Enrolmen
 
   /**
    * @description
-   * Check for the requirement of at least one certification, or one job.
+   * Check for the requirement of at least one certification, or one obo site/job.
    */
   public hasCertificateOrJob(): boolean {
-    const jobs = this.jobsForm.get('jobs') as FormArray;
+    const oboSites = this.jobsForm.get('oboSites') as FormArray;
     const certifications = this.regulatoryFormState.certifications;
+    const enrolleeHealthAuthorities = this.careSettingsForm.get('enrolleeHealthAuthorities') as FormArray;
+    let hasOboSiteForEveryHA = true;
+    // Using `for` loop rather than `every()` method for ease of debugging
+    for (let i = 0; i < enrolleeHealthAuthorities.controls.length; i++) {
```

Comment:
With JS being functional there isn't much need to do full `for` loop syntax and if you're not using `every` use `forEach`.  I would suggest `every` if that is the intent since you have a comment explaining why you're not using `every`, but I don't see how not using it eases debugging.

Reply:
I'll make these updates quickly in a separate PR